KnownIssuesService/Services/KnownIssuesService.cs

```diff
@@ -0,0 +1,175 @@
+		{
+			var credentials = Authenticate();
+			WorkItemTrackingHttpClient workItemTrackingHttpClient = new WorkItemTrackingHttpClient(new Uri(_knownIssuesUri), credentials);
+
+			return workItemTrackingHttpClient;
+		}
+
+		/// <summary>
+		/// Creates a WorkItemQuery Result used for getting work item ids and their urls from 
+		/// Azure DevOps Org
+		/// </summary>
+		/// <returns>WorkItem Query result</returns>
+		public async Task<WorkItemQueryResult> GetQueryByWiqlAsync()
+		{
+			WorkItemQueryResult result = await _httpQueryClient.QueryByWiqlAsync(_workItemQuery, null, 100, null, CancellationToken.None).ConfigureAwait(false);
+
+			return result;
+		}
+
+		/// <summary>
+		/// Retrieves WorkItems data from Azure DevOps Org based on the passed in work item ids
+		/// </summary>
+		/// <param name="ids">Work Item Ids From Azure DevOps Organisation</param>
+		/// <param name="result">WorkItemQuery result that contains the specific ids and their urls</param>
+		/// <returns>a List of Work Items from Azure DevOps Org</returns>
+		public async Task<List<WorkItem>> GetWorkItemsQueryAsync(int[] ids, WorkItemQueryResult result)
+		{
+			List<WorkItem> items = await _httpQueryClient.GetWorkItemsAsync(ids, null, result.AsOf, null, null, null, CancellationToken.None).ConfigureAwait(false);
+			return items;
+		}
+
+		/// <summary>
+		/// Create a Query builder for retrieving work items from an Azure DevOps Organisation
+		/// </summary>
+		/// <returns>a work item query builder containing the selection criteria</returns>
+		public Wiql QueryBuilder()
+		{
+			// create a wiql object and build our query
+			Wiql wiql = new Wiql()
+			{
+				Query = "Select [Id] " +
+						"From WorkItems " +
+						"Where [Work Item Type] = 'Bug' " +
+						"And [System.TeamProject] = '" + UtilityConstants.knownIssuesOrganisation + "' " +
+						"Order By [State] Asc, [Changed Date] Desc",
+			};
+			return wiql;
+		}
+
+		/// <summary>
+		/// Function to Query the List of Known Issues from Azure DevOps Known Organization
+		/// </summary>
+		/// <returns>Known Issues Contract that contains json items that will be rendered on the browser</returns>
+		public async Task<List<KnownIssuesContract>> QueryBugs()
+		{
+			List<KnownIssuesContract> knownIssuesList = new List<KnownIssuesContract>();
+			try
+			{
+				WorkItemQueryResult result = await GetQueryByWiqlAsync();
+				var ids = result.WorkItems.Select(item => item.Id).ToArray();
+
+				if (ids.Length == 0)
+				{
+					return new List<KnownIssuesContract>();
+				}
+
+				// get work items for the ids found in query
+				var test = result.AsOf;
+				List<WorkItem> items = await GetWorkItemsQueryAsync(ids, result);
+
+				foreach (var item in items)
+				{
+					KnownIssuesContract contract = new KnownIssuesContract();
+					contract.Id = item?.Id;
+
+					if (item.Fields.ContainsKey("System.State"))
+					{
+						contract.State = item.Fields["System.State"].ToString();
```

Comment:
```suggestion
						contract.State = item.Fields.TryGetValue("System.State", out var state) ? state.ToString() : default;
```
And remove the surrounding if block
for all the instances below as well